fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentLiveTestsTemplate.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * // Copyright (c) Microsoft Corporation. All rights reserved.
+ * // Licensed under the MIT License.
+ */
+
+package com.azure.autorest.fluent.template;
+
+import com.azure.autorest.fluent.model.clientmodel.*;
+import com.azure.autorest.model.javamodel.JavaFile;
+import com.azure.autorest.util.CodeNamer;
+import com.google.common.collect.Lists;
+
+public class FluentLiveTestsTemplate {
+
+    private static final FluentLiveTestsTemplate INSTANCE = new FluentLiveTestsTemplate();
+
+    public static FluentLiveTestsTemplate getInstance(){
+        return INSTANCE;
+    }
+
+    public void write(FluentLiveTests liveTests, JavaFile javaFile) {
+        // write class
+        addImports(liveTests, javaFile);
+        javaFile.publicClass(Lists.newArrayList(), liveTests.getClassName(), classBlock->{
+            for (FluentLiveTestCase testCase : liveTests.getTestCases()) {
+                // write manager field
+                classBlock.privateMemberVariable(liveTests.getEntryType().getName(), liveTests.getEntryName());
+                // write @BeforeEach
+                classBlock.annotation("BeforeEach");
+                classBlock.publicMethod("void init()", methodBlock -> {
```

Comment:
Use the overloaded `setup()` in `TestBase`.

Reply:
Added.